types/underscore/index.d.ts

```diff
@@ -5758,177 +5758,177 @@ declare module _ {
         * Wrapped type `object`.
         * @see _.omit
         **/
-        omit(...keys: string[]): _Chain<T>;
-        omit(keys: string[]): _Chain<T>;
-        omit(iteratee: Function): _Chain<T>;
+        omit(...keys: string[]): _Chain<T, V>;
+        omit(keys: string[]): _Chain<T, V>;
```

Comment:
I wonder whether it would be possible to do this more accurately, something like this:

```ts
omit<U extends string>(keys: U[]): _Chain<Omit<T, V[U]>, Omit<V, >>;
```

But this is probably beyond the scope of this pull request and it might require a TS upgrade, so definitely not a condition for acceptance.

Reply:
Yeah if @regevbr is contemplating a complete refactor anyway, I would rather hold off.

Reply:
indeed we should hold off on that one